Tribler/Main/vwxGUI/list_body.py

```diff
@@ -763,6 +763,9 @@ def OnCollapse(self, item=None, raise_events=True, from_expand=False):
 
     @warnWxThread
     def OnChange(self, scrollToTop=False):
+        if GUIUtility.getInstance().utility.abcquitting:
```

Comment:
List_body should be a generic component, no need for a call to guiutility here.
A PyDeadObject can be fixed by doing
```
if self:
```
no need for abcquitting